cockroachdb/tests/conftest.py

```diff
@@ -13,16 +13,23 @@
 
 
 @pytest.fixture(scope='session', autouse=True)
-def dd_environment(instance):
+def dd_environment(instance, instance_e2e):
```

Comment:
why include both here?

Reply:
We're still using url from the main instance for verification of successful up

Reply:
Can you elaborate on why?

Reply:
Actually, this doesn't prevent port collisions b/c the files still expose ports. I'm closing this